hwdb/70-mouse.hwdb

```diff
@@ -507,4 +507,4 @@ mouse:usb:v145fp01ac:name:HID-compliant Mouse Trust Gaming Mouse:
 
 # Zelotes 5500 DPI 7 Button USB Wired Gaming Mouse
 mouse:usb:v1d57pad17:*
- MOUSE_DPI=*1000@500 1600@500 2400@500 3200@500 5500@500
+ MOUSE_DPI=1000@500 1600@500 2400@500 3200@500 5500@500 *1000@1000 1600@1000 2400@1000 3200@1000 5500@1000
```

Comment:
is the report rate software-controlled or is it a physical hardware switch? If the latter, we can add it. In the former case we don't add them unless the 1000Hz is the out-of-the-box setting (in which case we'd have to remove the 500Hz)